charts/metallb/templates/podmonitor.yaml

```diff
@@ -84,6 +85,8 @@ rules:
       - ""
     resources:
       - pods
+      - services
```

Comment:
I don't think we do need services / endpoints in case of podmonitor, pods are enough